docs/outputs.md

```diff
@@ -3372,6 +3372,166 @@ This directory contains outputs from the Crossborder model, which represents all
 </table>
 
 
+
+### CVM Establishment Synthesis model outputs (.\cvmestsynthesis)
+
+
+<table>
+  <tr>
+   <td>File 
+   </td>
+   <td>Description
+   </td>
+  </tr>
+  <tr>
+   <td>MGRAEmpByEstSize.csv
+   </td>
+   <td>MGRA based synthetically generated establishments. Used for diagnostic purposes, not for simulation
+   </td>
+  </tr>
+  <tr>
+   <td>SummarySynthEstabs.csv
+   </td>
+   <td>Total count summary by size class and industry sector
+   </td>
+  </tr>
+  <tr>
+   <td>SynthEstablishments.csv
+   </td>
+   <td>Contains information about synthetically generated establishments to be used as inputs to the commercial vehicle model 
+   </td>
+  </tr>
+  
+</table>
+
+#### CVM Establishment Synthesis Model Trip File ( MGRAEmpByEstSize.csv )
+
+<table>
+  <tr>
+    <td>Field
+    </td>
+    <td>Description
+    </td>
+  </tr>
+  <tr>
+    <td>mgra
+    </td>
+    <td>MGRA id
+    </td>
+  </tr>
+  <tr>
+    <td>taz
+    </td>
+    <td>TAZ id
+    </td>
+  </tr>
+  <tr>
+    <td>luz
+    </td>
+    <td>Land use zone
+
+  </tr>
+  <tr>
+    <td>emp_Sector Size_Class
+    </td>
+    <td>Employment numbers, where Sector is gov, mil, ag_min, bus_svcs, fin_res_mgm, educ, hlth, ret, trn_wrh, con, utl, mnf, whl, ent, accm, food, oth; where Size_Class ranges 1-7.
+
+   
+
+</table>
+
+#### CVM Establishment Synthesis Model Trip File ( SummarySynthEstabs.csv )
+
+<table>
+  <tr>
+    <td>Field
+    </td>
+    <td>Description
+    </td>
+  </tr>
+  <tr>
+    <td>Industry_no
+    </td>
+    <td>Industry No
+    </td>
+  </tr>
+  <tr>
+    <td>Industry_Name
+    </td>
+    <td>Industry name
+    </td>
+  </tr>
+  <tr>
+    <td>Size_Class
+    </td>
+    <td>Size class
+  </td>
+
+  </tr>
+  <tr>
+    <td>Count
+    </td>
+    <td>Total number
+    </td>
+    </tr>
+
+</table>
+
+#### CVM Establishment Synthesis Model Trip File ( SynthEstablishments.csv )
+
+<table>
+  <tr>
+    <td>Field
+    </td>
+    <td>Description
+    </td>
+  </tr>
+  <tr>
+    <td>Establishment_ID
+    </td>
+    <td>Unique establishment id (index) for each establishment
+    </td>
+  </tr>
+  <tr>
+    <td>Industry_No
+    </td>
+    <td>The category of industry that the establishment belongs to (range: 1-12)
+    </td>
+  </tr>
+  <tr>
+    <td>Industry_Name
+    </td>
+    <td>The name of the industry that the establishment belongs to (EPO, AGM, CON 
+etc.)
+    </td>
+
+  </tr>
+  <tr>
+    <td>LUZ
+    </td>
+    <td>Land Use Zone (Or the TAZ) to which the establishment belongs to
+  </td>
+  </tr>
+  <tr>
+    <td>MGRA
+    </td>
+    <td>The MGRA number to which the establishment belongs to
+  </td>
+  </tr>
+  <tr>
+    <td>Employees
+    </td>
+    <td>The number of employees in the establishment
+  </td>
+  </tr>
+  <tr>
+    <td>Size_Class
+    </td>
+    <td>The categorized size of the establishment based on the number of employees
+  </td>
+  </tr>
+</table>
+
```

Comment:
@mansisandag Please modify from line #3375 to line#3533. Those files are inputs for CVM model (see input/CVM). Please move those to input page.